docs/guide/cache/content_aware_cache.md

```diff
@@ -293,3 +350,253 @@ bin/console fos:httpcache:invalidate:tag ez-all
 
     Similarly to purging from code, the tags you purge on, are prefixed to match the currently configured SiteAccess. 
     When you use this command in combination with multi-repository setup, make sure to specify SiteAccess argument.
+
+## Testing and Debugging HTTP cache
+
+It is important to test your code in an environment which is as similar as your production environment as possible. That
+means that if only are testing locally using the default Symfony Reverse proxy when your are going to use Varnish or
+Fastly in production, you are likely edning up some (bad) surprises. Due to the symfony reverse proxy's lack of support for ESIs, it behaves
+quite different from Varnish and Fastly in some aspects.
+If you are going to use Varnish in production, make sure you also test your code with Varnish.
+If you are going to use Fastly in production, testing with Fastly in your developer install is likely not feasible
+(you're local development environment must then be accessible for Fastly). Testing with Varnish instead will in most
+cases do the job. But if you need to change the varnish configuration to make your site work, be aware that Varnish and Fastly uses different dialects, and
+that .vcl code for Varnish V6.x will likely not work as-is on Fastly.
```

Comment:
```suggestion
You should test your code in an environment similar to your production environment.
	If you only test locally using the default Symfony reverse proxy when you want to use Varnish or
	Fastly in production, you may encounter problems. Symfony reverse proxy does not support ESI, it behaves differently than Varnish and Fastly.
	If you want to use Varnish in production, make sure you also test your code with Varnish.
	If you want to use Fastly in production, testing with Fastly in your developer installation is likely not feasible
	(local development environment must be then accessible for Fastly). Testing with Varnish instead works in most	cases.
	But if you need to change the Varnish configuration to make your site work, be aware that Varnish and Fastly use different dialects, and
	that .vcl code for Varnish 6.x may not work as it does in Fastly.
```

Reply:
In the last sentence here.. now you say it may not work in varnish in the same way "as it does in Fastly", right? It is the opposite of my original intention

Reply:
I corrected the last sentence.